hw2/04-background.js

```diff
@@ -0,0 +1,48 @@
+// Add your code here
+document.addEventListener("DOMContentLoaded", () => {
+  const toggleButton = document.getElementById("toggle");
+  const intervalInput = document.getElementById("interval");
+  let backgroundColorInterval;
+  let isStarted = false;
+
+  // reference: https://www.rustcodeweb.com/2021/07/how-to-generate-random-rgb-color-using-javascript.html
+  const randomColor = () => {
+    const x = Math.floor(Math.random() * 256);
+    const y = Math.floor(Math.random() * 256);
+    const z = Math.floor(Math.random() * 256);
+    const a = 0.2;
+    const RGBColor = "rgb(" + x + "," + y + "," + z + "," + a + ")";
```

Comment:
According to the Airbnb Style, we should use template literals for string concatenation. For example, when generating the RGB color string, use template literals:

`const RGBColor = `rgb(${x},${y},${z},${a})`;`

Reply:
Thanks for pointing out Fixed it!